packages/react-native/Libraries/Text/RCTTextAttributes.m

```diff
@@ -130,9 +130,12 @@ - (NSParagraphStyle *)effectiveParagraphStyle
 
   if (!isnan(_lineHeight)) {
     CGFloat lineHeight = _lineHeight * self.effectiveFontSizeMultiplier;
-    paragraphStyle.minimumLineHeight = lineHeight;
-    paragraphStyle.maximumLineHeight = lineHeight;
-    isParagraphStyleUsed = YES;
+    // text with lineHeight lower then font.lineHeight does not correctly vertically align
```

Comment:
Can you capitalize all comments and add a line break before them (except for the one after the if which is in a new level of indentation)?
```suggestion
    // Text with lineHeight lower then font.lineHeight does not correctly vertically align
```

Reply:
Thanks @cead22. I made the changes with commit https://github.com/Expensify/react-native/pull/57/commits/12b51ddb8b995e4a08278d5094c6d2c838c09f04. I remain available for further improvements.